src/uniprotkb/components/protein-data-views/__tests__/__snapshots__/XrefView.spec.tsx.snap

```diff
@@ -1,1134 +1,6 @@
-              >
-                1234
-                <test-file-stub
-                  data-testid="external-link-icon"
-                  width="12.5"
-                />
-              </a>
-              [
-              <a
-                href="#ABCD"
-              >
-                ABCD
-              </a>
-              ]
-            </li>
-          </ul>
-        </div>
-      </div>
-    </li>
-  </ul>
-</DocumentFragment>
-`;
-
-exports[`XRefView should render for Structure section 1`] = `
-<DocumentFragment>
-  <h3 />
-  <ul
-    class="info-list info-list--columns"
-  >
-    <li>
-      <div
-        class="decorated-list-item"
-      >
-        <div
-          class="decorated-list-item__title"
-        >
-          <h5
-            class="bold"
-          >
-            EMBL
-          </h5>
-        </div>
-        <div
-          class="decorated-list-item__content"
-        >
-          <ul
-            class="expandable-list no-bullet"
-          >
-            <li>
-              (
-              <a
-                class="external-link"
-                href="https://www.ebi.ac.uk/ena/data/view/1234"
-                rel="noopener"
-                target="_blank"
-              >
-                EMBL
-                <test-file-stub
-                  data-testid="external-link-icon"
-                  width="12.5"
-                />
-              </a>
-               | 
-              <a
-                class="external-link"
-                href="https://www.ncbi.nlm.nih.gov/protein/1234"
-                rel="noopener"
-                target="_blank"
-              >
-                GenBank
-                <test-file-stub
-                  data-testid="external-link-icon"
-                  width="12.5"
-                />
-              </a>
-               | 
-              <a
-                class="external-link"
-                href="http://getentry.ddbj.nig.ac.jp/getentry/dad/1234"
-                rel="noopener"
-                target="_blank"
-              >
-                DDBJ
-                <test-file-stub
-                  data-testid="external-link-icon"
-                  width="12.5"
-                />
-              </a>
-              ) 
-              <a
-                class="external-link"
-                href="//www.ebi.ac.uk/ena/data/view/1234"
-                rel="noopener"
-                target="_blank"
-              >
-                1234
-                <test-file-stub
-                  data-testid="external-link-icon"
-                  width="12.5"
-                />
-              </a>
-              [
-              <a
-                href="#ABCD"
-              >
-                ABCD
-              </a>
-              ]
-            </li>
-          </ul>
-        </div>
-      </div>
-    </li>
-  </ul>
-</DocumentFragment>
-`;
-
-exports[`XRefView should render for SubCellularLocation section 1`] = `
+exports[`XRefView should render section 1`] = `
```

Comment:
I remember you asking about this test, just want to check that the drop in lines in the snapshot is expected

Reply:
Yep, no drop in coverage as it was testing the same thing in a loop. All the removed snapshots are identical

Reply:
> Looks good, I think we should meet to discuss best practices and document it in the coding guidelines (tagging @dlrice ). Some I've noticed in your PR:
> 
> * `jest.fn()` instead of `()=>{}`
> * `screen` instead of find*/get* from `render()`
> * hierarchy of react-testing-library selectors
> * `test.each()`
>   but there must be more to discuss in https://kentcdodds.com/blog/common-mistakes-with-react-testing-library

All looks good to me